content/player/guide/game_translation.md

```diff
@@ -20,18 +22,36 @@ The translation function in EasyRPG Player is similar to variant two, but is eas
 
 ### First Steps
 
-First create a new folder called ``languages`` in your game folder. In this folder you can now create further subfolders (one for each translation). The folder name does not matter. In the following we will call the folder ``LANGUAGE_NAME``.
+First create a new folder called ``Languages`` in your game folder. In this folder you can create further subfolders (one for each translation). The folder name does not matter. In the following we will call the folder ``LANGUAGE_NAME``.
```

Comment:
If I'm not mistaken the "Languages" folder was renamed to "Language" in the latest continuous builds, so you may also have to change that in the guide.
![screenshot_43](https://user-images.githubusercontent.com/108797333/210137671-7dfef79d-decb-4fcd-a27b-e6e7777482df.png)

Reply:
Also mentioned at lines 102, 104 and 152.